recipes-core/packagegroups/packagegroup-core-x11-sato.bbappend

```diff
@@ -0,0 +1,6 @@
+
+NETWORK_MANAGER:pinenote-a55 = "networkmanager"
+
+# PineNote is using NetworkManager, which conflicts with connman that is used by sato
+#RDEPENDS_${PN}-base:remove:pinenote-a55 += "connman-gnome connman"
+#RDEPENDS_${PN}-base:remove:pinenote-a55 += "${NETWORK_MANAGER}"
```

Comment:
Do we need these?

Reply:
No, I don't think so, will do a bit of cleanup of commented out stuff.